### ISSUE TITLE
Roadmap for next release ?

### ISSUE THREAD

**Issue by kavink** [JanusGraph/janusgraph#942]
I was wondering where can I track when will there be a next release and the changes to expect/roadmap for next release. I see last release was in Oct but could not find any doc with roadmap or similar.

**Comment by twilmes** [comment]
Thanks for your interest in JanusGraph @kavink. Release planning is currently underway. You can follow along on the dev list:

[0.3.0](https://groups.google.com/d/topic/janusgraph-dev/0ng5lHio4TQ/discussion)
[0.2.1](https://groups.google.com/d/topic/janusgraph-dev/BGaCCH1cTCE/discussion)

We do not maintain a road map document at this point. The best place to track what is going on in the project is to keep an eye on the open issues and which Github milestones they are assigned to.

**Comment by Textras** [comment]
Was wondering the same thanks! Btw came from this rather interesting comment back and forth;

"I have deployed JanusGraph on AWS but in slightly more elegant way - using ECS. Briefly, it is an ECS cluster running JG containers (Gremlin server, connecting to DynamoDB), fronted with Application Load Balancer (can potentially support sticky sessions, but I have not tested that yet). Then a similar set-up for ElasticSearch with ECS. This part I do not like at all - I would love to use AWS ElasticSearch service, but JG needs scripting and it is only available for ES 5. So, I am waiting for new JG release which is supposed to support ES 5+ - this way I can simplify my configuration big time.

 
Reply
−
Avatar
Paul T  ngrigoriev
3 days ago
Has this occured yet? Would love a tutorial on your setup. Thanks for the tips

 
EditReply
−
Avatar
ngrigoriev  Paul T
3 days ago
Yeah, I should write one. Briefly: yes, it works. We have contributed some code to JG (not yet released but integrated in the master) that supports AWS ES service, with proper auth, HTTPS etc. So, our set-up was really simple: AWS ES service as indexing backend, DynamoDB using https://github.com/awslabs/..., Gremlin servers on the ECS containers and an ELB to from them. Once next release of JG is available, it will be much easier.

Source link:
